pkg/cmd/pr/create/create.go

```diff
@@ -122,7 +122,7 @@ func NewCmdCreate(f *cmdutil.Factory, runF func(*CreateOptions) error) *cobra.Co
 	return cmd
 }
 
-func createRun(opts *CreateOptions) error {
+func createRun(cmd *cobra.Command, opts *CreateOptions) error {
```

Comment:
To be able to do `cmd.Flags().Changed(name string)` I ended up introducing an extra parameter, which is what broke the tests.

Reply:
@msfjarvis Add 2 more fields, `TitleProvided` and `BodyProvided` to `CreateOptions`, and access them via `opts` then

titleProvided and bodyProvided are calculated in lines 84-85 also

Reply:
Thanks, I'll update this shortly